### PR TITLE
Add descriptions for Synth SA fields, create and update

### PR DESCRIPTION
**Pull request by b3nk4n**
# Why/What

Add more detailed descriptions and CURL examples to our API docs for Synthetic Smart Alerts.

# References

- https://instana.kanbanize.com/ctrl_board/37/cards/116652/details/
- https://github.ibm.com/instana/backend/pull/17519

### CODE REVIEW

**Review by ediazgallego — APPROVED** [review]
Looks good to me, thanks!

File: spec/descriptions/createSyntheticTest.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ This API endpoint creates a Synthetic Test.
 
 ```
 curl -k -v -X POST \
-http://<hostname>/api/synthetics/settings/tests \
--H 'authorization: apiToken valid-api-token' \
+https://<Host>/api/synthetics/settings/tests \
+-H 'authorization: apiToken <Token>' \
```
Comment (b3nk4n) [inline spec/descriptions/createSyntheticTest.md:9]: Just for consistency with placeholders used in other CURL examples.

File: spec/descriptions/updateSyntheticAlertConfig.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Use this API endpoint to update a Synthetic Smart Alert by ID. Effectively, it creates a new revision instead of replacing the previous configuration.
+
+#### Example
+
+```bash
+curl -k --request POST \
+  --url https://<Host>/api/events/settings/global-alert-configs/synthetics/<ID> \
+  --header 'authorization: apiToken <Token>' -H 'Content-Type: application/json' \
+  --data '{
+      "id":"<ID>",
```
Comment (carolzhang65) [inline spec/descriptions/updateSyntheticAlertConfig.md:10]: Question: Is id needed? It is part of the URL path.

File: spec/descriptions/tagSyntheticAlertConfiguration.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+The API endpoints of this group can be used to manage alert configurations for Synthetic Monitoring.
+
+## Parameters:
+
+- **id:** ID of the alert configuration.
```
Comment (carolzhang65) [inline spec/descriptions/tagSyntheticAlertConfiguration.md:5]: Maybe add a note that the id is generated when creating a Synthetic alert.

**Pull request merged**